folding-schemes/src/folding/nova/decider_eth_circuit.rs

```diff
@@ -562,10 +563,16 @@ where
         poseidon_config: &PoseidonConfig<C::ScalarField>,
         U_i: CommittedInstance<C>,
     ) -> Result<(C::ScalarField, C::ScalarField), Error> {
-        let (cmE_x_limbs, cmE_y_limbs): (Vec<C::ScalarField>, Vec<C::ScalarField>) =
-            point_to_nonnative_limbs::<C>(U_i.cmE)?;
-        let (cmW_x_limbs, cmW_y_limbs): (Vec<C::ScalarField>, Vec<C::ScalarField>) =
-            point_to_nonnative_limbs::<C>(U_i.cmW)?;
+        let (cmE_x_limbs, cmE_y_limbs, _): (
+            Vec<C::ScalarField>,
+            Vec<C::ScalarField>,
+            Vec<C::ScalarField>,
+        ) = point_to_nonnative_limbs::<C>(U_i.cmE)?;
+        let (cmW_x_limbs, cmW_y_limbs, _): (
```

Comment:
Is there a reason to ignoring the flag indicating if this is the infinity point when computing challenges?

Reply:
Yes, because we're already identifying them as [0,1] when infinity point, so was ignoring the 'infinity' boolean. But changed mind regarding the whole 'infinity' bit/boolean, and I think that we can directly remove it from the point representation. Updated the PR to reflect that. This will also reduce the needed amount of constraints.